src/Andre/SoulsFormats/SoulsFormats/Formats/DCX.cs

```diff
@@ -109,14 +109,18 @@ internal static Memory<byte> Decompress(BinaryReaderEx br, out Type type)
                     byte unk30 = br.GetByte(0x30);
                     byte unk38 = br.GetByte(0x38);
 
+                    // Credit to ivi
+                    if (BinaryReaderEx.IsFlexible && unk04 != 0x11000)
+                        unk04 = 0x10000;
```

Comment:
Hmmm, this selection logic is tricky.
Since all the changes of dflt type are just asserts, is it saner to just do if / else if / else type = Type.DCX_DFLT_UNKNOWN, and let that brick at the asserts (excluding that, with flexible on, the asserts will pass and there will be no issue)
This way we don't introduce the flexibility logic directly into the format... though we do instead make up some DFLT type, however aptly named.
Certainly hiding the leniency from the format definition is tricky in this case, so if anyone else has any thoughts feel free to weigh in, it's the only thing making me hesitant at this point.